sources/core/Stride.Core.Yaml.Tests/DescriptorTests.cs

```diff
@@ -232,7 +224,7 @@ public void TestDictionaryDescriptor()
         {
             var attributeRegistry = new AttributeRegistry();
             var factory = new TypeDescriptorFactory(attributeRegistry);
-            var descriptor = new DictionaryDescriptor(factory, typeof(Dictionary<int, string>), false,
+            DictionaryDescriptor descriptor = new DictionaryDescriptor(factory, typeof(Dictionary<int, string>), false,
```

Comment:
Unnecessary change, not a big deal, but if you have time

Reply:
reverted that